presto-native-execution/presto_cpp/main/common/Counters.cpp

```diff
@@ -47,6 +47,7 @@ void registerPrestoMetrics() {
       kCounterHttpClientNumConnectionsCreated, facebook::velox::StatType::SUM);
   DEFINE_METRIC(kCounterNumQueryContexts, facebook::velox::StatType::AVG);
   DEFINE_METRIC(kCounterNumTasks, facebook::velox::StatType::AVG);
+  DEFINE_METRIC(kCounterBytesProcessed, facebook::velox::StatType::SUM);
```

Comment:
While we talked about using SUM, can we check the corresponding JMX metric and see if it is also representing a sum and is increasing over time. It might be an AVG to indicate the "throughput" amount of bytes processed on a worker.